pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -1166,14 +1166,16 @@ type BackupSpec struct {
 	ServiceAccount string `json:"serviceAccount,omitempty"`
 	// CleanPolicy denotes whether to clean backup data when the object is deleted from the cluster, if not set, the backup data will be retained
 	CleanPolicy CleanPolicyType `json:"cleanPolicy,omitempty"`
+	// TableFilter means Table filter expression for 'db.table' matching
```

Comment:
We have to make it clear that BR only supports this from v4.0.3.

Reply:
https://github.com/pingcap/tidb-operator/issues/2695#issuecomment-655865350
Yes it is.

Reply:
```suggestion
	// TableFilter means Table filter expression for 'db.table' matching. BR supports this from v4.0.3.
```